tests/rules/no-missing-atx-heading-space.test.js

```diff
@@ -0,0 +1,423 @@
+/**
+ * @fileoverview Tests for no-missing-atx-heading-space rule.
+ * @author Sweta Tanwar (@SwetaTanwar)
+ */
+
+//------------------------------------------------------------------------------
+// Imports
+//------------------------------------------------------------------------------
+
+import rule from "../../src/rules/no-missing-atx-heading-space.js";
+import markdown from "../../src/index.js";
+import { RuleTester } from "eslint";
+import dedent from "dedent";
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester({
+	plugins: {
+		markdown,
+	},
+	language: "markdown/commonmark",
```

Comment:
```suggestion
	language: "markdown/gfm",
```

Could you change it to `gfm`? If you're trying to use `strikethrough` in your test case, you'll need to set the language to `gfm` since it's a GFM-only syntax.